ui/src/main/java/de/ipb_halle/lbac/admission/UserMgrBean.java

```diff
@@ -375,7 +375,7 @@ public void setModeUpdate() {
      * whitespace!
```

Comment:
Please some more detail -> whitespaces are not trimmed because of ldab problematic